src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.LogicApps.tests.ps1

```diff
@@ -1,382 +1,167 @@
-                }
-                Mock Get-AzCachedAccessToken -MockWith {
-                    return @{
-                        SubscriptionId = "123456"
-                        AccessToken = "accessToken"
-                    }
-                }
-
-                # Act
-                Disable-AzLogicAppsFromConfig -DeployFileName "$PSScriptRoot\Files\deploy-orderControl-noWaitingOrRunningRunsWithImmediate.json" -ResourceGroupName $resourceGroup
-
-                # Assert
-                Assert-MockCalled Get-AzCachedAccessToken -Scope It -Times 1
-                Assert-MockCalled Get-AzLogicAppRunHistory -Scope It -Times 10 -ParameterFilter { $ResourceGroupName -eq $resourceGroup }
-                Assert-MockCalled Disable-AzLogicApp -Scope It -Exactly 5 -ParameterFilter { $resourceGroupName -eq $resourceGroup }
             }
-            It "Disables all logic apps when checkType = NoWaitingOrRunningRuns with found waiting and running runs for stopType = Immediate" {
+        }
+        Context "Enabling Logic Apps with configuration" {
+            It "Enable logic app when stopType = Immediate" {
                 # Arrange
-                $resourceGroup = "my-resource-group"
-                $i = 0
-                Mock Get-AzLogicAppRunHistory {
-                    $ResourceGroupName | Should -Be $resourceGroup
-                    $Name | Should -Be "snd-async"
-                    $script:i++
-                    if ($script:i -gt 2) {
-                        Write-Host "Returning empty (no running, no waiting) runs"
-                        return @()
-                    } else {
-                        Write-Host "Returning 1 running & 1 waiting runs"
-                        return @(
-                            [pscustomobject]@{ Status = "Running" },
-                            [pscustomobject]@{ Status = "Waiting" }
-                        )
-                    }
-                }
-                Mock Disable-AzLogicApp {
-                    $ResourceGroupName | Should -Be $resourceGroup
-                    $LogicAppName | Should -Be "snd-async"
-                }
-                Mock Get-AzCachedAccessToken -MockWith {
-                    return @{
-                        SubscriptionId = "123456"
-                        AccessToken = "accessToken"
-                    }
+                $resourceGroupName = $config.Arcus.ResourceGroupName
+                $logicAppName = Create-AzLogicAppName
+               
+                $filePath = [System.IO.Path]::GetTempFileName()
+                $json = '[
+                  {
+                    "description": "Sender(s)",
+                    "checkType": "None",
+                    "stopType": "Immediate",
+                    "logicApps": [
+                      "' + $logicAppName + '"
+                    ]
+                  }
+                ]'
+                Set-Content $filePath $json
```

Comment:
This trick is to make sure that we don't have to check-in additional files in source control. With this, we can generate a Logic App workdefinition in a temp file and remove that temp file after the test has run.